helix-rest/src/main/java/org/apache/helix/rest/server/service/InstanceService.java

```diff
@@ -90,8 +89,22 @@ InstanceInfo getInstanceInfo(String clusterId, String instanceName,
    * Get the current instance stoppable checks
    * @param clusterId The cluster id
    * @param instanceName The instance name
+   * @param jsonContent The json payloads from client side
    * @return An instance of {@link StoppableCheck} easily convertible to JSON
+   * @throws IOException in case of network failure
    */
   StoppableCheck getInstanceStoppableCheck(String clusterId, String instanceName,
       String jsonContent) throws IOException;
+
+  /**
+   * Batch get StoppableCheck results for a list of instances in one cluster
+   *
```

Comment:
Doesn't helix-style.xml remove extra lines in block comments? I am not sure if this PR has been formatted with helix-style.xml, although it says so in the PR description.